### PR TITLE
VB-2450 Update visit schedule for HMP Morton Hall (MHI) in VSIP

### PR DESCRIPTION
**Pull request by AledWEvans**
VB-2450
Update visit schedule for HMP Morton Hall (MHI) in VSIP

### CODE REVIEW

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

**Review by hutcheonb-moj — APPROVED** [review]
LGTM - session template references to be added to the second SQL script after running first

File: src/main/resources/db.scripts.mvp/update_visit_times_mhi.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+-- This sql makes sure visits in future aline up to the latest session times and session templates.
+
+UPDATE visit
```
Comment (dhirajshettymoj) [inline src/main/resources/db.scripts.mvp/update_visit_times_mhi.sql:3]: do we need to do this? the ticket says  - Make this live effectively immediately - any booked visits will only have a minor impact

File: src/main/resources/db.scripts.mvp/update_visit_times_mhi.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+FROM (SELECT v.* FROM visit v
+    LEFT JOIN legacy_data ld ON ld.visit_id = v.id
+    JOIN prison p ON p.id = v.prison_id
+    WHERE p.code = 'MHI' AND v.visit_start > CURRENT_DATE AND ld.id IS NULL) AS tmp
+WHERE tmp.id = visit.id and UPPER(TRIM(To_Char(tmp.visit_start , 'DAY'))) = 'TUESDAY';
+
+UPDATE visit
+    SET visit_start = date_trunc('day', tmp.visit_start) + '14:00:00',
+    visit_end = date_trunc('day', tmp.visit_end) + '16:00:00',
+    session_template_reference = <PLEASE NETER ST REF FOR THURSDAY>
```
[the remark this answers is not in the document]
Comment (AledWEvans) [reply src/main/resources/db.scripts.mvp/update_visit_times_mhi.sql:16]: yes,

**Pull request merged**